x-pack/filebeat/input/entityanalytics/provider/okta/conf.go

```diff
@@ -27,6 +27,7 @@ func defaultConfig() conf {
 		SyncInterval:   24 * time.Hour,
 		UpdateInterval: 15 * time.Minute,
 		LimitWindow:    time.Minute,
+		LimitFixed:     nil,
```

Comment:
This isn't needed. Is it here for explicitness?

Reply:
I guess I was thinking out loud. Doesn't need to be there. Removed.